src/Avalonia.Controls/Notifications/IManagedNotificationManager.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) The Avalonia Project. All rights reserved.
+// Licensed under the MIT license. See licence.md file in the project root for full license information.
+
+namespace Avalonia.Controls.Notifications
+{
+    /// <summary>
+    /// Interface for a managed notification manager.
+    /// Managed notification managers can show any content.
+    /// </summary>
```

Comment:
```suggestion
    /// </summary>
    /// <remarks>
    /// Because notification managers of this type are implemented purely in managed code, they
    /// can display arbitrary content, as opposed to notification managers which display notifications
    /// using the host operating system's notification mechanism.
    /// </remarks>
```